packages/core/src/extensions/Blocks/nodes/Block.ts

```diff
@@ -292,13 +311,16 @@ export const Block = Node.create<IBlock>({
           }
 
           const { startPos, contentNode } = blockInfo;
+          const contentTypeName = typeof type !== "string" ? type.name : type;
+          const contentTypeAttrs =
```

Comment:
This means type attrs will always be optional right? So for example, just passing "heading" will be valid, and we'll probably set it to heading level 1?

Reply:
Yeah the headings and list items have default attributes, they're defined in `Heading.ts` and `ListItem.ts` (level: 1 for headings and ordered: false for list items).